src/main/java/com/cglx/Calculator.java

```diff
@@ -1,5 +1,6 @@
 package com.cglx;
 
+
 public class Calculator {
 
     public Calculator() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.io.FileReader'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-01:9990/coding_rules#rule_key=squid%3AUselessImportCheck)